master/usage/troubleshooting/index.md

```diff
@@ -18,6 +18,10 @@ or you can set environment variables for `sudo` commands like this:
     sudo ETCD_ENDPOINTS=http://172.25.0.1:2379 calicoctl node
```

Comment:
shouldn't this be `... node run` since you're showing an example of a command? (`calicoctl node` isn't really a complete command anymore)